docs/toc.html

```diff
@@ -111,22 +117,28 @@
                         <li><a href="#multitenancy-georeplication">Multi-Tenancy and Geo-Replication</a></li>
                         <li><a href="#multitenancy-more">Further considerations</a></li>
                     </ul>
+                </li>
                 <li><a href="#config">6.5 Important Configs</a>
                     <ul>
                         <li><a href="#clientconfig">Important Client Configs</a>
                         <li><a href="#prodconfig">A Production Server Configs</a>
                     </ul>
+                </li>
                 <li><a href="#java">6.6 Java Version</a>
                 <li><a href="#hwandos">6.7 Hardware and OS</a>
                     <ul>
-                        <li><a href="#os">OS</a>
-                        <li><a href="#diskandfs">Disks and Filesystems</a>
-                        <li><a href="#appvsosflush">Application vs OS Flush Management</a>
-                        <li><a href="#linuxflush">Linux Flush Behavior</a>
-                        <li><a href="#ext4">Ext4 Notes</a>
+                        <li><a href="#os">OS</a></li>
+                        <li><a href="#diskandfs">Disks and Filesystems</a></li>
+                        <li><a href="#appvsosflush">Application vs OS Flush Management</a></li>
+                        <li><a href="#linuxflush">Linux Flush Behavior</a></li>
+                        <li><a href="#filesystems">Filesystem Selection</a></li>
+                        <li><a href="#replace_disk">Replace KRaft Controller Disk</a></li>
                     </ul>
+                </li>
                 <li><a href="#monitoring">6.8 Monitoring</a>
                     <ul>
+                        <li><a href="#remote_jmx">Security Considerations for Remote Monitoring using JMX</a></li>
+                        <li><a href="#kraft_monitoring">KRaft Monitoring Metrics</a></li>
```

Comment:
I think we can remove `Metrics` for consistency with the ones below

Reply:
Yes. Removed metrics from the title.